src/interactive/Modal.jsx

```diff
@@ -129,6 +133,19 @@ class Modal extends React.Component {
 			</div>
 		);
 
+		const closeArea = (
+			<div className='padding--all'>
```

Comment:
this could solve the left alignment problem with the close button in a hero (pass an arg `hasHero` or something)

```
const closeArea = hasHero => (
	<div className={cx(
		'padding--all',
		{ 'flush--left': hasHero }
	)}>
	...
);
```